runtime-sdk/modules/evm/src/types.rs

```diff
@@ -116,3 +116,24 @@ mod eth {
     impl_upstream_conversions!(H160, H256, U256);
 }
 pub use eth::{H160, H256, U256};
+
+#[derive(Clone, Copy, Default, Eq, PartialEq)]
+pub struct StdU64 {
+    data: u64,
```

Comment:
Is there a reason for this field? It seems like you only ever use `bytes`?

Reply:
Yes，this field is redundant. I'll delete it